gfx/portraits/portrait_modifiers/05_headgear_situational.txt

```diff
@@ -231,4 +231,45 @@
 			}
 		}
 	}
+
+	# Warcraft
+	special_metaljaw = {
+		dna_modifiers = {
+			accessory = {
+				mode = add
+				gene = special_metaljaw
+				template = metaljaw
+				range = { 0 1 }
+				value = 0.6
+			}
+		}
+		outfit_tags = { metal_jaw }
+		weight = {
+			base = 0
+			modifier = {
+				add = 5
+				has_trait = being_undead
+			}
+			modifier = {
+				factor = 0
+				has_trait = creature_high_elf
```

Comment:
trait check can be replaced by `culture ?= { has_cultural_pillar = heritage_highborne }`